.travis.yml

```diff
@@ -2,19 +2,22 @@ language: php
 
 sudo: false
 
+php:
+  - 7.1
+  - 7.2
+  - 7.3
+  - nightly
```

Comment:
you can remove nightly because the dependencies are not ready for php 8

Reply:
sorry I was thinking nightly is still for 7.4 but now I checked it and changed